scripts/src/lib/generator/templates/entrypoint/Entrypoint.kt.eta

```diff
@@ -0,0 +1,8 @@
+package <%= it.package %>
+
+@Suppress("unused")
+fun init() {
```

Comment:
Why not a mod initializer class/singleton here 🤔 (also the logger would be a nice addition here as well, could be a top-level property even)

Reply:
Yes, I copied this from the first example I found on github. Ideally this should match the java one as closely as possible.